llvm/test/MC/AArch64/win-import-call-optimization-no-section.s

```diff
@@ -0,0 +1,9 @@
+// RUN: not llvm-mc -triple aarch64-windows-msvc -filetype obj < %s 2>&1 | FileCheck %s
+
+tail_call:
+  adrp    x8, __imp_b
+  ldr     x8, [x8, :lo12:__imp_b]
+  .impcall        __imp_b
+  br     x8
+
+// CHECK: error: .impcall directives were used, but no existing .impcall section exists
```

Comment:
Is it possible to include a source location

Reply:
Unfortunately, no: the `.impcall` section may appear after the `.impcall` directives, so I'd need to keep track of the locations of all (or at least one) observed `.impcall` directives, which is not easy to do with the way asm parsing vs object writing is setup.